modules/python/Makefile.security

```diff
@@ -0,0 +1,15 @@
+BANDIT=$(call which, bandit)
+BANDIT_FLAGS=-ll -ii
+
+PYTHON_LINT_TARGETS?=$(shell find . -name "*.py" -not -path "*/.venv/*" -not -path "*/$(BUILD_HARNESS_VENV_NAME)/*" -not -path "*/.tox/*" -not -path "*/.egg*" -not -path "*/frontend/*" -not -path "*/dist/*" -not -path "*/build/*" -not -path "*/tests/*")
```

Comment:
PYTHON_LINT_TARGETS is already set, so you should just be able to remove this line altogether

Reply:
I was excluding tests from the security linting whereas styling linting hits those files.

Reply:
AH! Ok, this was the answer to my original question. In that case, it makes sense to set this to `PYTHON_SECURITY_TARGETS`. We should move all the find flags that are shared into a variable too. Something like 

```makefile
PYTHON_TARGETS_FIND_FLAGS=-name "*.py" -not -path "*/.venv/*" -not -path "*/$(BUILD_HARNESS_VENV_NAME)/*" -not -path "*/.tox/*" -not -path "*/.egg*" -not -path "*/frontend/*" -not -path "*/dist/*" -not -path "*/build/*"

PYTHON_LINT_TARGETS?=$(shell find . $(PYTHON_TARGETS_FIND_FLAGS))
PYTHON_SECURITY_TARGETS?=$(shell find . $(PYTHON_TARGETS_FIND_FLAGS) -not -path "*/tests/*")
```